src/main/java/io/druid/data/input/MapBasedRow.java

```diff
@@ -19,12 +18,11 @@
 public class MapBasedRow implements Row
 {
   private static final Logger log = new Logger(MapBasedRow.class);
-
+  private static final Function<Object, String> TO_STRING_INCLUDING_NULL = new ToStringFunction();
```

Comment:
Instead of making a subclass of Function can you just instantiate here?

Doing so is more consistent with other places in the druid source.